python/ray/_raylet.pyx

```diff
@@ -1300,31 +1287,27 @@ cdef report_streaming_generator_output(
             context.caller_address,
             context.task_id,
             context.return_size,
-            context.generator_index,
+            generator_index,
             context.is_async,
             &return_obj)
-        # Del output here so that we can GC the memory
-        # usage asap.
-        del output_or_exception
 
-        context.streaming_generator_returns[0].push_back(
-            c_pair[CObjectID, c_bool](
-                return_obj.first,
-                is_plasma_object(return_obj.second)))
+    # Del output here so that we can GC the memory
+    # usage asap.
+    del output_or_exception
 
-        logger.debug(
```

Comment:
Can you keep the log here?

Reply:
Do we really need it? Doesn't seem to be adding a lot of value (I'm adding some debug logs in C++ core)

Reply:
Hmm yeah let's remove it then if we have cpp logs. I added all logs for myself when I debugged in the past haha